### ISSUE TITLE
Intake Take2 is about to release

### ISSUE THREAD

**Issue by martindurant** [intake/intake-thredds#136]
Please consider pinning intake<2 to keep this package working, if it breaks with the new code.

**Comment by martindurant** [comment]
It should be possible to port existing users to intake.readers.catalogs.THREDDSCatalogReader

**Comment by tlvu** [comment]
Any plan to make this `intake-thredds` compatible with the newer `intake` 2?

If we want `intake-xarray` 2, we need `intake` 2, but then this `intake-thredds` is not compatible with `intake` 2.

**Comment by martindurant** [comment]
Good point - I'll look into it. Intake 2 does have a THREDDSCatalogReader internally, so it should not be much work (hopes).
